src/Features/VisualBasic/Portable/VBFeaturesResources.resx

```diff
@@ -1235,4 +1235,7 @@ Sub(&lt;parameterList&gt;) &lt;statement&gt;</value>
   <data name="Make_Async_Sub" xml:space="preserve">
     <value>Make Async Sub</value>
   </data>
+  <data name="Multiple_Types" xml:space="preserve">
```

Comment:
Why is this here as well?  What is it needed for?  If it is needed, we def should not have the same string in two different projects.